mailserver.env

```diff
@@ -419,6 +418,26 @@ ENABLE_GETMAIL=0
 # The number of minutes for the interval. Min: 1; Max: 30.
 GETMAIL_POLL=5
 
+# -----------------------------------------------
+# --- OAUTH2 Section ----------------------------
+# -----------------------------------------------
+
+# empty => OAUTH2 authentication is disabled
+# 1 => OAUTH2 authentication is enabled
+ENABLE_OAUTH2=
+
+# empty => verySecretId
+# Specify the OAuth2 client ID
+OAUTH2_CLIENT_ID=
+
+# empty => verySecretSecret
+# Specify the OAuth2 client secret
+OAUTH2_CLIENT_SECRET=
+
+# empty => https://oauth2.example.com/userinfo/
+# Specify the user info endpoint URL of the oauth2 provider. The trailing slash is MANDATORY (at least for Authentik)
+OAUTH2_INTROSPECTION_URL=
+
```

Comment:
This section needs to be added here as well: <https://docker-mailserver.github.io/docker-mailserver/latest/config/environment/>.